uiowa/uiowa01
# Updated #212: Added/updated RSS feeds on relevant entity_types.

Added new RSS feeds for events and seminars. These may not be perfect, but they are something until a better solution is created. Updated RSS feeds for articles and deadlines to match new design.

Testing is rather straightforward:

- Enable all entity_types_* modules.
- Create new entities for articles, deadlines, events, and seminars.
- Navigate to /articles.xml, /deadlines.xml, /events.xml, and /seminars.xml to see if the feeds are valid.

NOTE: You will have to turn off Twig debugging to get valid feeds.
